package.json

```diff
@@ -211,7 +211,7 @@
 		"@types/wordpress__api-fetch": "^3.2.2",
 		"@types/wordpress__block-editor": "^2.2.8",
 		"@types/wordpress__block-library": "^2.6.0",
-		"@types/wordpress__components": "^8.5.4",
+		"@types/wordpress__components": "^9.8.0",
```

Comment:
Any concern from the gutenboarding point of view from updating this? Should we test anything before shipping? cc @Automattic/luna

Reply:
I think a quick smoke test is to see if anything breaks on `/new`. Current link: https://hash-829c18b8bcfddd5cf96dd3c4bfb03dd0b6df309f.calypso.live/new

Reply:
Type issues should fail loudly in CI anyway, no?

Reply:
It shouldn't be necessary to smoke test anything given it's just type definition updates.